suite-native/biometrics/src/components/AuthenticatorProvider.tsx

```diff
@@ -1,21 +1,12 @@
 import React, { ReactNode } from 'react';
 
-import { BiometricOverlay } from './BiometricOverlay';
-import { useIsBiometricsEnabled } from '../biometricsAtoms';
-import { useBiometrics } from '../useBiometrics';
-
 type AuthenticatorProviderProps = {
     children: ReactNode;
 };
 
-export const AuthenticatorProvider = ({ children }: AuthenticatorProviderProps) => {
-    const { isUserAuthenticated } = useBiometrics();
-    const { isBiometricsOptionEnabled } = useIsBiometricsEnabled();
-
-    return (
-        <>
-            {children}
-            {isBiometricsOptionEnabled && !isUserAuthenticated && <BiometricOverlay />}
-        </>
-    );
-};
+export const AuthenticatorProvider = ({ children }: AuthenticatorProviderProps) => (
+    <>
+        {children}
+        {/* {shouldShowBiometricOverlay && isBiometricsOptionEnabled && <BiometricOverlay />} */}
```

Comment:
We don't want overlay anymore?

Reply:
We do, this will be added in follow up PR as discussed before.